CMake/HPHPFindLibs.cmake

```diff
@@ -464,14 +458,7 @@ macro(hphp_link target)
     target_link_libraries(${target} double-conversion)
   endif()
 
-  if (LZ4_FOUND)
-    target_link_libraries(${target} ${LZ4_LIBRARY})
-  else()
-    target_link_libraries(${target} lz4)
-  endif()
-  # The syntax used for these warnings is unparsable by Apple's Clang
-  add_definitions("-DLZ4_DISABLE_DEPRECATE_WARNINGS=1")
```

Comment:
Mac build failed for an unrelated reason, I'll re-run it to test whether this is still needed.